tests/scripts/bulk_export.py

```diff
@@ -0,0 +1,128 @@
+"""
```

Comment:
Since this script is very similar to `bulk_import.py`, it would be ideal if there could be one script that performed both functions.  Obviously the script would have to be renamed, maybe `bulk_import_export.py`

Reply:
yes or performance.py :)